docs/pages/access-controls/guides/device-trust.mdx

```diff
@@ -0,0 +1,326 @@
+
+__Device enrollment__ is performed either by a device admin or by the end-user,
+at your discretion. This is the step that creates the Secure Enclave private key
+in the device and register its public key counterpart with the Teleport Auth
+Server. Enrollment has to run in the actual device that you want to enroll. For
+example, this happens when a user gets a new device for the first time, or when
+IT prepares a new device for a user. Enrollment only needs to happen once per
+user/device combination.
+
+<Admonition type="tip" title="Enrollment">
+  Device enrollment is key in correctly identifying a device throughout its
+  lifetime. The more trusted the enrollment operator, the better are the ongoing
+  guarantees that the device itself is trustworthy.
+</Admonition>
+
+The tie between the device inventory and enrollment is a device enrollment
+token. A device enrollment token needs to be created by a device admin and sent
+to the person performing the enrollment ceremony. The token ties the actual
+device being enrolled to the expected device in the inventory, adding an extra
+layer of protection.
+
+## Prerequisites
+
+(!docs/pages/includes/commercial-prereqs-tabs.mdx!)
+
+- A signed and notarized `tsh` binary for enrollment.
+  [Download the macOS tsh installer](../../installation.mdx#macos).
+- A macOS device to register and enroll.
+
+## Step 1/2. Register a trusted device
+
+The `tctl` tool is used to manage the device inventory. A device admin is
+responsible for managing devices, adding new devices to the inventory and
+removing devices that are no longer in use.
+
+A role that grants permissions for the `device` resource is necessary to manage
+the inventory. Save the yaml below as `device-admin.yaml` and create it in your
+cluster:
+
+```yaml
+version: v6
+kind: role
+metadata:
+  name: device-admin
+spec:
+  allow:
+    rules:
+    - resources: ["device"]
+      verbs:
+      - create
+      - read
+      - list
+      - update
+      - delete
+      - create_enroll_token
+      - enroll
+```
+
+```code
+$ tctl create -f device-admin.yaml
+# role 'device-admin' has been created
+```
+
+<Admonition type="tip" title="Editor Role">
+  For clusters created using Teleport v12 or newer, the builtin role `editor`
+  has the necessary permissions to manage devices.
+  For production setups a separate role is still recommended.
+</Admonition>
+
+Note that, in addition to the usual CRUD verbs (create, read, list, update and
+delete), we have also included `create_enroll_token` and `enroll`. The
+`create_enroll_token` verb is necessary to execute the `tctl devices enroll`
+command; `enroll` is necessary to execute `tsh device enroll`. Both are
+demonstrated in the following section.
+
+Make sure your user has the `device-admin` role:
+
+```yaml
+kind: user
+metadata:
+  name: codingllama
+  # unrelated fields omitted.
+spec:
+  roles:
+  - access
+  - device-admin # add this line
+version: v2
+```
+
+```code
+$ tctl edit users/codingllama
```

Comment:
First use of `tctl edit` in the docs! 🎉

Reply:
`tctl edit` is too good. I wish we could `tctl edit` resources that don't exist yet...  ;)